frontend/model/contracts/group.js

```diff
@@ -1412,8 +1466,7 @@ sbp('chelonia/defineContract', {
   methods: {
     'gi.contracts/group/_cleanup': ({ contractID, state }) => {
       // unsubscribe from other group members identity contract
-      const identityContractID = sbp('state/vuex/state').loggedIn?.identityContractID
-      const possiblyUselessContractIDs = Object.keys(state.profiles || {}).filter(cID => cID !== identityContractID)
```

Comment:
Why was this change made?

I remember it used to be this way, and `identityContractID` was added to prevent some bug.

Reply:
>  was added to prevent some bug.

That bug should be fixed instead.

> Why was this change made?

If the reference count is correct, this code is simpler. There is no need not to remove ourselves because there should still be a positive reference count for our own identity contract. OTOH, if the reference count is wrong, that bug should be fixed. This PR is supposed to correct those errors.

Reply:
Are we sure that the reference count on our own contractID then is in fact correct in this case?